### PR TITLE
Improve instructions for installing VS2017 and building for ARM

### PR DESCRIPTION
**Pull request by BruceForstall**

### CODE REVIEW

**Comment by BruceForstall** [comment]
cc @dotnet/jit-contrib @dotnet/arm32-contrib

**Comment by CarolEidt** [comment]
Would it make sense to also add explicit instructions for the build command to use? Perhaps it should be obvious, but it's certainly not obvious when cross-building for arm64 or for arm/linux from x64/linux?

Either here: https://github.com/dotnet/coreclr/blob/master/Documentation/building/cross-building.md

Or just inline where you've added this info?

File: Documentation/building/windows-instructions.md [hunk 1]
```diff
@@ -45,7 +45,8 @@ For Visual Studio 2017:
     * Visual Studio C++ core features
   * Under "SDKs, libraries, and frameworks":
     * Windows 10 SDK or Windows 8.1 SDK
-* To build for Arm32, Make sure that you have the Windows 10 SDK installed (or selected to be installed as part of VS installation). To explicitly install Windows SDK, download it from here: [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads)
+* To build for Arm32, Make sure that you have the Windows 10 SDK installed (or selected to be installed as part of VS installation). To explicitly install Windows SDK, download it from here: [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads).
```
Comment (sandreenko) [inline Documentation/building/windows-instructions.md]: There are two separate boxes for arm and arm64.

Comment (RussKeldorph) [reply Documentation/building/windows-instructions.md]: Probably best not to mention ARM64 for windows unless/until the build is supported.

**Comment by BruceForstall** [comment]
Added a section on building ARM for Windows in cross-building.md

**Comment by CarolEidt** [comment]
Thanks @BruceForstall !!

**Pull request merged**